src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Providers/Providers/IaasVmPsBackupProvider.cs

```diff
@@ -280,6 +285,121 @@ public RecoveryPointBase GetRecoveryPointDetails()
         }
 
         /// <summary>
+        /// Provisioning Item Level Recovery Access for the given recovery point
+        /// </summary>
+        /// <returns>Azure VM client script details as returned by the service</returns>
+        public RPMountScriptDetails ProvisionItemLevelRecoveryAccess()
+        {
+            string content = string.Empty;
+            AzureVmRecoveryPoint rp = ProviderData[RestoreBackupItemParams.RecoveryPoint]
+                as AzureVmRecoveryPoint;
+            if (rp.EncryptionEnabled == true)
```

Comment:
@dragonfly91 do we need to check if `rp` is null before accessing a property of it?

Reply:
This case would never happen as we have [ValidateNotNullOrEmpty] declared on top of the RecoveryPoint parameter in the cmdlet definition. So, I don't think a null check is needed.